### PR TITLE
View topics in collection

### PR DESCRIPTION
**Pull request by Mentessi**

### CODE REVIEW

File: app/models/topic.rb [hunk 1]
```diff
@@ -2,4 +2,7 @@ class Topic < ApplicationRecord
   validates :name, presence: true
 
   belongs_to :user
+  # has_and_belongs_to_many :collections
+  has_many :collection_topic_memberships
+  has_many :topics, through: :collection_topic_memberships
```
Comment (Neilos) [inline app/models/topic.rb]: Think this is left over from our chat.
You shoulnd't (can't?) mix _:has_and_belongs_to_many_ relationships with _:has_many_ and _:has_many through_ relationships. Use one of the other.

File: db/migrate/20180503074809_create_join_table_collection_topic.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class CreateJoinTableCollectionTopic < ActiveRecord::Migration[5.1]
+  def change
+    create_join_table :collections, :topics do |t|
+      # t.index [:collection_id, :topic_id]
+      # t.index [:topic_id, :collection_id]
```
Comment (Neilos) [inline db/migrate/20180503074809_create_join_table_collection_topic.rb]: Why are these indexes commented out?
Indexes help the database find things more quickly.

File: spec/models/collection_spec.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Collection, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (Neilos) [inline spec/models/collection_spec.rb]: Generally I either delete the file or delete the pending line. I use pending specs to remind myself I have work to do on the current feature. Leaving a pending spec implies that things are incomplete.
As you will almost certainly be writing specs for a collection in the near future, I'd just delete the pending line.

File: spec/system/managing_collections_spec.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe 'managing collections' do
+  let!(:collection) { FactoryBot.create(:collection, name: 'dogs and cats') }
+  let!(:topic){ FactoryBot.create(:topic, name: 'dog topic') }
+  let!(:unrelated_topic){ FactoryBot.create(:topic, name: 'unrelated') }
```
Comment (Neilos) [inline spec/system/managing_collections_spec.rb]: _:unrelated_topic_ - nice. Very declarative and intention revealing.

Comment (Mentessi) [reply spec/system/managing_collections_spec.rb]: haha... this was the test you wrote for me ;)

**Pull request merged**